elements/lisk-transactions/test/base_transaction.ts

```diff
@@ -501,6 +504,15 @@ describe('Base transaction class', () => {
 			expect(status).to.eql(Status.OK);
 		});
 
+		it('should return a successful transaction response with a valid transaction with basic impl', async () => {
+			transactionWithBasicImpl.sign('passphrase');
```

Comment:
shouldn't we check if we add the invalid fee, it becomes `0`?

Reply:
I've added a new test case scenario ;)